Modules/Test/classes/class.ilTestScoringByQuestionsGUI.php

```diff
@@ -154,44 +143,108 @@ protected function showManScoringByQuestionParticipantsTable($manPointsPost = ar
         $table->setTitle($this->lng->txt('tst_man_scoring_by_qst'));
 
         if ($selected_questionData) {
-            $maxpoints = assQuestion::_getMaximumPoints($selected_questionData['question_id']);
+            $maxpoints = assQuestion::_getMaximumPoints((int) $selected_questionData['question_id']);
             $table->setCurQuestionMaxPoints($maxpoints);
-            $maxpoints = ' (' . $maxpoints . ' ' . $this->lng->txt('points') . ')';
+            $maxpoints_txt = ' (' . $maxpoints . ' ' . $this->lng->txt('points') . ')';
             if ($maxpoints == 1) {
-                $maxpoints = ' (' . $maxpoints . ' ' . $this->lng->txt('point') . ')';
+                $maxpoints_txt = ' (' . $maxpoints . ' ' . $this->lng->txt('point') . ')';
             }
 
             $table->setTitle(
-                $this->lng->txt('tst_man_scoring_by_qst') . ': ' . $selected_questionData['title'] . $maxpoints .
+                $this->lng->txt('tst_man_scoring_by_qst') . ': ' . $selected_questionData['title'] . $maxpoints_txt .
```

Comment:
Additionally I found an issue here: `$maxpoints` has been overwritten immediately.

Reply:
This pattern appears multiple times within this class. To improve the code through refactoring even further, i would suggest extracting this section into a separate function.

```php
    private function formatPoints(float $max_points): string
    {
        return sprintf('(%d %s)', $max_points, $this->lng->txt($max_points == 1 ? 'point' : 'points'));
    }
```

Alternatively, I suggest using a ternary operator instead of introducing an additional variable.
If this change is desired, i'm happy to do this refactoring.